content/blog/browser-extension-recommendations.en.md

```diff
@@ -0,0 +1,66 @@
+    "tags": [ "privacy", "online", "ads", "cookies", "https" ],
+    "authors": [ "nishantwrp" ]
+}
+
+Websites these days collect a lot of information about you. When you leave the website, they then want to know where you are going next. They leave tracking cookies in your browser. 
+
+Tools like Google Analytics, they register everything from your geographical location to how often to visit a webpage.
+
+**Below are some privacy add-ons that give your browser extra muscles to keep you safe online.** 
+
+## Privacy Badger
+
+<img src="/img/blog/online-privacy/privacy-badger.png">
+
+[Privacy Badger](https://www.eff.org/privacybadger) is arguably the best anti-tracking web browser extension. Launched by Electronic Frontier Foundation (EFF), it’s a lightweight add-on with a user-friendly toolbar button that shows the detected trackers on the page. The “green/yellow/red” block-list system shows which webpages are tracking you and gives you the ability to block any or all of it at any point. The one feature that sets Privacy Badger apart is that it doesn’t block all content, just the tracking scripts that follow you from one site to another. If the content tracking you is essential to a page’s functionality, Privacy Badger will allow it to load with the tracking capabilities blocked.
+
+Privacy Badger is available for [Firefox](https://addons.mozilla.org/en-US/firefox/addon/privacy-badger17/), Chrome, Opera and even Firefox on Android phones.
+
+## uBlock Origin
+
+<img src="/img/blog/online-privacy/ublock-origin.png">
+
+uBlock Origin is one of the most popular ad and content blocker extensions. It prevents pesky popups and widgets from sending your data to advertisers. uBlock Origin lets you create personalized lists which gives you more control over what ads to block and what ads to allow. It has also been found to effectively block more ads than others while using less memory, so it doesn’t slow down your browsing experience.
+
+uBlock Origin is available for [Firefox](https://addons.mozilla.org/en-US/firefox/addon/ublock-origin/), Chrome, Opera and Safari.
+
+## HTTPS Everywhere
+
+<img src="/img/blog/online-privacy/https.jpg">
+
+Secure websites use the HTTPS (Hyper Text Transfer Protocol Secure) protocol, which encrypts your credit card details, personal information, and other sensitive data. However, not all websites are HTTPS-enabled.
+
+[HTTPS Everywhere](https://www.eff.org/https-everywhere) add-on is a joint venture project of Tor browser and EFF. When you install the extension, it automatically rewrites the requests, establishes a connection to SSL, then locates and directs you to the most secure versions of the websites you visit. In case your browser connects you to unsecured websites, HTTPS Everywhere will keep your data private and secure from leaking by redirecting it to an encrypted HTTPS.
```

Comment:
```suggestion
[HTTPS Everywhere](https://www.eff.org/https-everywhere) add-on is a joint venture project of the Tor browser and EFF. When you install the extension, it automatically rewrites the requests, establishes a connection via TLS, then locates and directs you to the most secure versions of the websites you visit. In case your browser connects you to unsecured websites, HTTPS Everywhere will keep your data private and secure from leaking by redirecting it to an encrypted HTTPS.
```